scanner/updater/export.go

```diff
@@ -66,11 +67,25 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 		bundles[uSet] = []updates.ManagerOption{updates.WithEnabled([]string{uSet})}
 	}
 
+	// Rate limit to ~16 requests/second by default.
+	interval := 62*time.Millisecond
+	configuredInterval := os.Getenv("STACKROX_SCANNER_V4_UPDATER_INTERVAL")
+	if configuredInterval != "" {
+		parsedInterval, err := time.ParseDuration(configuredInterval)
+		switch {
+		case err != nil:
+			log.Println(fmt.Errorf("invalid interval, using default (%v): %w", interval, err))
+		case parsedInterval < interval:
+			log.Println(fmt.Errorf("interval is too small (%v): using default (%v)", parsedInterval, interval))
```

Comment:
nit: wouldn't `log.Printf` have the same output? (vs creating the intermediary error)

```suggestion
			log.Printf("invalid interval, using default (%v): %v", interval, err)
		case parsedInterval < interval:
			log.Printf("interval is too small (%v): using default (%v)", parsedInterval, interval)
```

Reply:
I chose `fmt.Errorf` for `%w`, but it is a bit messy. I'll make this change